docs/license.rst

```diff
@@ -9,4 +9,4 @@ Ccdproc License
 
 Ccdproc is licensed under a 3-clause BSD style license:
 
-.. include:: ../licenses/LICENSE.rst
+.. include:: ../LICENSE.rst
```

Comment:
hm, is it a good idea to `include` the license file? Wouldn't `literalinclude` make more sense?